shadowsocks/stream.go

```diff
@@ -141,7 +140,9 @@ type Reader interface {
 // NewShadowsocksReader creates a Reader that decrypts the given Reader using
 // the shadowsocks protocol with the given shadowsocks cipher.
 func NewShadowsocksReader(reader io.Reader, ssCipher shadowaead.Cipher) Reader {
-	return &shadowsocksReader{reader: reader, ssCipher: ssCipher}
+	return &readConverter{
+		r: &shadowsocksReader{reader: reader, ssCipher: ssCipher},
```

Comment:
Maybe rename `shadowsocskReader` to `shadowsocskChunkReader`, since it's not a `Reader`. Or maybe just `chunkReader` so it doesn't stutter.

Reply:
Done